src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp

```diff
@@ -38,7 +48,8 @@ void InternetConnectionService::setupChannel()
 
     try
     {
-        channel_ = AmqpClient::Channel::Create(ipAddress_.toStdString(), port_);
+        //channel_ = AmqpClient::Channel::Create(ipAddress_.toStdString(), port_);
```

Comment:
I don't think there's any reason to leave this line commented out - if we need to revert, we should probably be looking at the commit history rather than comments.